rpm_spec/subpackages/manageiq-core

```diff
@@ -49,12 +48,12 @@ done
 %files core
 %defattr(-,root,root,-)
 %{app_root}
-%config(noreplace) %{app_root}/certs
```

Comment:
no replace is only for files and not a directive for directories
We also have no files in this directory that would be overwritten.